composer.json

```diff
@@ -49,9 +54,9 @@
         "sonata-project/core-bundle": "^3.8",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.1.11",
-        "symfony/phpunit-bridge": "^4.2",
-        "symfony/routing": "^2.8 || ^3.2",
+        "symfony-cmf/testing": "^2.1.8",
```

Comment:
Why did you downgrade that one???

Reply:
sorry, this was old, before your changes or i fucked it up in my rebase